### DISCUSSION TITLE
Mac Catalyst support

### DISCUSSION THREAD

**Discussion by marcosrdz** [lodev09/react-native-true-sheet#58] [Ideas]
True Sheet modal covers the whole app screen when being used with Mac Catalyst

**Comment by lodev09** [comment]
I don't really have experience with Mac Catalyst... I did try it on iPad and it works ok.. Do you have an example I can use?

**Reply by marcosrdz** [reply]
Not atm but it literally covers the whole current screen with no way to dismiss unless u have a button for it inside the true sheet. For Mac Catalyst im guessing an anchored popover on the source view would be the expected behavior

**Reply by lodev09** [reply]
Can you send screenshot? It might be default behavior on Mac apps. I read it [here](https://stackoverflow.com/questions/72871811/tap-outside-to-dismiss-modally-presented-view-in-mac-catalyst).
